docs/CONFIG-VARS.md

```diff
@@ -72,7 +72,7 @@ Tool uses the jump server to interact with rwx filestore, that needs to be pre-m
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_MANAGE_STORAGE | Should the tool manage the storageclass | bool | true | false | Set to false if you wish to manage the storage class | all |
+| V4_CFG_MANAGE_STORAGE | Should the tool manage the storageclass | bool | false | false | Set to false if you wish to manage the storage class | all |
```

Comment:
this is a pretty big change. Lots of folks depend on this being set to `true`. We'll need a really good reason to break things for folks who have external database and don't care to manager their storage.

Reply:
It's actually not a behavior change but I'd agree with you if it was, it's only a doc update to reflect that the default in our code is really false instead of true. I 
am advocating for only updating the doc to reflect current code behavior and NOT changing the underlying default in the code here ->https://github.com/sassoftware/viya4-deployment/blob/276de04e4e664b59c8ff790a8763998d1e3b2787/roles/vdm/defaults/main.yaml#L28

Our input files in examples/ seem to include "V4_CFG_MANAGE_STORAGE: true" which could result in that variable being set to true often unless they intentionally set it to false.
Check out the comment I made in IAC-788 which argues that updating  the doc is the least disruptive approach of the two so that our doc and code are in sync.

Reply:
Per team discussion this morning I've restored CONFIG-VARS.md so that the default indicated in the table for V4_CFG_MANAGE_STORAGE is true and I've updated the vdm defaults main.yaml file, line 28 so that the default value for V4_CFG_MANAGE_STORAGE is now true.